### PR TITLE
Use absolute paths in the extractor

### PR DESCRIPTION
**Pull request by herschel666**
@HenrikFricke PTAL :octocat:

### CODE REVIEW

**Comment by herschel666** [comment]
refs https://github.com/Jimdo/web-templates/issues/991

File: src/collector/extractor.coffee [hunk 1]
```diff
@@ -9,30 +9,31 @@ assets = require '../../config/assets.js'
 
 class Collector
   constructor: (@read_fn, @design_id, @image_folder = false) ->
+    @cwd = @cwd
```
Comment (nesQuick) [inline src/collector/extractor.coffee]: does this line make any sense?

Comment (herschel666) [inline src/collector/extractor.coffee]: no, multiple selection shot me in the knee 😩

**Comment by HenrikFricke** [comment]
@nesQuick should give you the goal for it, he already comment the code :)

**Comment by herschel666** [comment]
Ich will nicht rum nerven, aber es wäre geil, wenn wir unser Ticket (Jimdo/web-templates#991) innerhalb dieser Iteration (endet morgen Abend) noch fertig kriegten. ;-)

**Comment by HenrikFricke** [comment]
@herschel666 in my perspective the change is absolutely valid, so merge it ;)

**Comment by herschel666** [comment]
@HenrikFricke I like!! :+1:

**Comment by nesQuick** [comment]
LGTM :)

**Pull request merged**